locales/he.po

```diff
@@ -399,7 +399,7 @@ msgstr "מחק את {0} ?"
 
 #: frontend/src/metabase/admin/datamodel/components/ObjectRetireModal.jsx:38
 msgid "Saved questions and other things that depend on this {0} will continue to work, but this {1} will no longer be selectable from the query builder."
-msgstr "{שאלות שמורות ופעולות אחרות שתלויות ב {0] ימשיכו לעבוד, אבל {1} לא יהיה זמין לבחירה במחולל הדוחות"
+msgstr "שאלות שמורות ופעולות אחרות שתלויות ב {0} ימשיכו לעבוד, אבל {1} לא יהיה זמין לבחירה במחולל הדוחות."
```

Comment:
Isn't this putting the period at the beginning of the sentence, since it's rtl?

Reply:
```suggestion
msgstr ".שאלות שמורות ופעולות אחרות שתלויות ב {0} ימשיכו לעבוד, אבל {1} לא יהיה זמין לבחירה במחולל הדוחות"
```

Punctuation goes at the end (left), like you can see in this example from a Hebrew newspaper: 
<img width="569" alt="image" src="https://github.com/metabase/metabase/assets/2223916/9a9e898d-8298-45bb-b371-c7ff8843b66a">

Reply:
My mind is just unable to parse multiline RTL. thanks!

![Screen Shot 2024-01-26 at 2 48 24 PM](https://github.com/metabase/metabase/assets/30528226/dc0ec987-418b-47d4-bb26-a74c21e434ee)